include/matx/generators/random.h

```diff
@@ -587,13 +497,24 @@ template <typename T, int RANK> class randomTensorView_t {
            std::enable_if_t<!std::is_array_v<remove_cvref_t<ShapeType>>, bool> = true>
   inline auto random(ShapeType &&s, Distribution_t dist, uint64_t seed = 0, LowerType alpha = 1, LowerType beta = 0)
   {
+    static_assert(
```

Comment:
Maybe use the second parameter to add a string saying why this is enforced.

Reply:
updated